internal/service/systemd.go

```diff
@@ -217,12 +210,20 @@ func (s *systemd) GetName() string {
 }
 
 func (s *systemd) reload() error {
-	return s.dbusConnection.ReloadContext(context.Background())
+	conn, err := newDbusConnection(s.Rootless)
```

Comment:
I would run defer `conn.close()` as always open the connection

Reply:
correct. forgot to add that. thanks